noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -157,12 +157,12 @@ impl PublicContext {
 
     // PrivateContextInputs must be temporarily passed in to prevent too many unknowns
```

Comment:
Some of this comment seems stale now no?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4648/commits/c57bb41775294b1d882bfb878185b545e58ac2c6